### DISCUSSION TITLE
关于作业11-planarity-coloring的一些小疑问

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#7] [Q&A] [answered]
作业第一题提到了补图的概念，我觉得没什么头绪，后面查询资料发现补图是要定义在完全图中的，请问老师是不是漏了加上G为完全图的前提条件呢...感觉加上这个条件才能够做出来并且用上顶点数 >= 11 的结论

**Comment by Martin7-1** [comment, marked as answer]
我思考了一下...好像是可以通过构建完全图，然后去掉图G来获得 \overline{G} 的...如果不加上完全图这个条件的话，好像G + \overline{G} 不一定为完全图，我们只要有一种不同于 G 将所有顶点连起来的方法就可以构建\obvrline{G}就可以了？

**Reply by Martin7-1** [reply]
最后一个是 \overline{G}...打错了

**Reply by hengxin** [reply]
@Martin7-1 是的, `G` 的补图 `\overline{G}`的概念是相对于完全图而言的。
